### PR TITLE
Fixes #73 #75

### LINKED ISSUE #73: Add new navbar to the layout
- [ ] inserir a barra do governo http://barra.governoeletronico.gov.br/ 
- [ ] uma barra com logo da fiocruz (exs: www.ensp.fiocruz.br/portal-ensp/ e www.ini.fiocruz.br) + EMAp-FGV;

Na pasta data há um svg com o logo da fiocruz e do procc já com as especificações da identidade visual da Fiocruz. Isso poderia ser incorporado no que hoje é a barra superior "Monitoramento de casos ..." ou na segunda barra, onde há os botões (como são de fundo branco, acho que ficará visualmente melhor aqui, na verdade).

(from @marfcg )

### PR DESCRIPTION
**Pull request by xmnlab**
@marfcg can you review this PR?

If you want some change we can treat that here.

### CODE REVIEW

File: fludashboard/templates/index.html [hunk 1]
```diff
@@ -15,10 +15,54 @@
       type="image/png"
       href="/static/img/favicon.ico"
   />
+  <style>
+    iframe {
+      padding: 0;
+      margin: 0;
+      border: 0;
+      width: 100%;
+      height: 110px;
```
Comment (marfcg) [inline fludashboard/templates/index.html:24]: height: 140px

File: fludashboard/templates/super-header.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+     <ul class="barra-fiocruz">
+        <li><a class="logo-fiocruz" href="http://www.fiocruz.br/" target="_blank"><span>Fiocruz</span></a></li>
+        <li><a class="fale-com" href="http://portal.fiocruz.br/pt-br/content/fale-conosco" target="_blank"><span>Fale com a Fiocruz</span></a></li>
+        <li><a class="logo-ensp" href="http://www.fiocruz.br/" target="_blank"><span>Fiocruz</span></a></li>
+    </ul>
+
+  </div>
+
+  <div style="margin-top: 10px; margin-bottom: 10px;">
+      <img src="/static/img/fgv-emap.png" alt="FGV-EMap" height="30px"/>
```
Comment (marfcg) [inline fludashboard/templates/super-header.html:47]: height="60px"

File: fludashboard/templates/super-header.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      <img src="/static/img/procc.png" alt="PROCC" height="30px" />
```
Comment (marfcg) [inline fludashboard/templates/super-header.html:48]: height="60px"

**Review by marfcg — CHANGES_REQUESTED** [review]
Great, @xmnlab !
Navbars and department logo's are working perfectly on the main page.

I've made a few specific suggestions to increase the size of the departments' logo.

A few additional comments:
- Help page doesn't keep the nav bars and logo;
- Help page is using the old version of README-pt_br, with "Implantação" and "Executando ..." sections;
- I've just realized that we never added the color code label to the map (check InfoDengue's page for reference: https://info.dengue.mat.br/). In our case it's (for both "detalhado" and "resumido"):
  - Atividade baixa (green)
  - Atividade epidêmica (yellow)
  - Atividade alta (orange)
  - Atividade muito alta (red)

PS: This help page is being generated reading which file?

Cheers!

**Comment by xmnlab** [comment]
@marfcg 

about your adicional comments, we can treat them in another PR.

I'm going to create the issues to treat them.

**Pull request merged**